Document pg_dump method for upgrading between major versions of Postgres on BigAnimal (#5342)
EnterpriseDB/docs: product_docs/docs/biganimal/release/using_cluster/05b_major_version_upgrade.mdx

---
title: Upgrading Postgres major versions
navTitle: Upgrading Postgres major versions
---

> To upgrade small to moderately large databases (up to several tens of gigabytes) between major versions of Postgres on BigAnimal, we recommend utilizing pg_dump to migrate data and populate a new BigAnimal instance/cluster. This method will require some downtime, so schedule the pg_dump operation accordingly.

Reviewer: Refer to https://www.enterprisedb.com/docs/biganimal/latest/migration/cold_migration/  topic to be aligned with font treatment for the utility names or any other keywords. We need to make the changes across the topic.

```suggestion
To upgrade small to moderately large databases (up to several tens of gigabytes) between major versions of Postgres on BigAnimal, we recommend utilizing `pg_dump` to migrate data and populate a new BigAnimal instance/cluster. This method requires downtime, so schedule the `pg_dump` operation accordingly.
```

> ## Migrating Postgres Versions with pg_dump

Reviewer: We use sentence case for the titles

```suggestion
## Migrating Postgres versions with pg_dump
```

!!! Important

> This procedure will likely accrue ingress and egress costs from your cloud service provider (CSP) for the migrated data. Please consult your CSP's pricing documentation to see how ingress and egress fees are calculated to determine any extra costs.

Reviewer: I think we can link to CSP's pricing documentation from here if that is available.

```suggestion
This procedure accrues ingress and egress costs from your cloud service provider (CSP) for the migrated data. Please consult your CSP's pricing documentation to see how ingress and egress fees are calculated to determine any extra costs.
```
